Sources/PackageLoading/PackageExtensions.swift

```diff
@@ -145,6 +145,15 @@ extension Package {
             }
         }
 
+        /// Check for targets that are not mapped to any modules
```

Comment:
That is right, `///` is a "documentation comment" which will be associated with the declaration, it should be used for documentation of functions and member variables, but local variables don't generally get documentation associated with them.